src/components/map/BoundingBoxControl.vue

```diff
@@ -19,23 +19,16 @@
 </template>
 
 <script setup lang="ts">
-import { computed } from 'vue'
-
-import { BoundingBox, boundingBoxToString } from '@/services/useBoundingBox'
+import { useBoundingBox } from '@/services/useBoundingBox'
 
 import DrawBoundingBoxControl from '@/components/map/DrawBoundingBoxControl.vue'
 
-const boundingBox = defineModel<BoundingBox | null>('boundingBox', {
-  default: null,
-})
+const { boundingBox, boundingBoxString } = useBoundingBox(1, 1)
+
```

Comment:
This creates a new bounding box state object, disconnecting the 
![image](https://github.com/Deltares/fews-web-oc/assets/144685504/e7406df2-11b3-4275-a748-c77539c15fd0)
bbox state and the 
![image](https://github.com/Deltares/fews-web-oc/assets/144685504/90ceb934-5283-4532-8a3a-2f56655ba15d)
bbox state.

The import used before was accessing global state.

Reply:
I have reverted the changes. I don't understand how the old import accessed global state, but I would like to know. I think I understand how the `const { boundingBox, boundingBoxString } = useBoundingBox(1, 1)` creates a new box (the `_boundingBox` instantiated in the `useBoundingBox` function. Is that correct?

Reply:
Not global state through import sorry,

Definemodel (a compiler macro for prop / emit definition) was accepting the bbox through a prop:
https://vuejs.org/guide/components/v-model.html

Reply:
Ahh, the boundingBox is being provided by the parent component as a `v-model`, using the `workflowsStore.boundingBox`. This does not provide reactivity to the lon/lat values in `BoundingBoxControl` when selecting a box.